eng/dashboardpack/dashboardpack.csproj

```diff
@@ -26,7 +26,7 @@
 
   <Target Name="Build" />
 
-  <Target Name="BeforeBuild" BeforeTargets="Build">
+  <Target Name="BeforeBuild" BeforeTargets="_GetPackageFiles">
```

Comment:
Why do we need this change? In general, it's best to avoid depending on "private" tragets (Targets that start with an underscore)

Reply:
I wanted to limit this target to only when we are generating a nuget.

Is this target needed only for generating the zip file? Is the zip file used for the dashboard docker image?

Reply:
This target mainly accomplishes two things:

- Adds the right content to the each of the dashboard packs (which actually may no longer be needed now that we have the dashboard as a tool) cc: @danegsta 
- Zips up the published output as that will be uploaded as a blob so that it can then be used to build the docker image for the dashboard.

Reply:
Reverted.